src/main/java/uk/gov/hmcts/reform/idam/client/IdamClient.java

```diff
@@ -13,10 +16,10 @@
 @Service
 public class IdamClient {
 
-    public static final String AUTH_TYPE = "code";
-    public static final String GRANT_TYPE = "authorization_code";
-    public static final String BASIC_AUTH_TYPE = "Basic";
-    public static final String BEARER_AUTH_TYPE = "Bearer";
+    private static final String AUTH_TYPE = "code";
```

Comment:
this is likely used outside the library, in places like mocking unit tests
please revert